content/en/docs/tasks/administer-cluster/encrypt-data.md

```diff
@@ -142,7 +142,8 @@ program to retrieve the contents of your secret.
     kubectl describe secret secret1 -n default
     ```
 
-    should match `mykey: mydata`
+    should match `mykey: bXlkYXRh`, mydata is encoded, check [decoding a secret](/docs/concepts/configuration/secret#decoding-a-secret) to
+    completely decode the secret.
 
```

Comment:
little confused about those change

Reply:
In docs,  we can consider that `mykey` and `mydata` are `var`, depend on `user`.

Reply:
even if we consider them as variable, the next step is decoding the secret to verify it is decrypted correctly.

Reply:
And it is inside task doc, so user follows the step by step tutorial

Reply:
```
kubectl describe secret secret1 -n default
```

The output contains `mykey: bXlkYXRh`. Note that the data, `bXlkYXRh` is base64 encoded.
[Decode the data](/docs/concepts/configuration/secret/#decoding-a-secret/), and you will get `mydata`.

Reply:
@steveperry-53 do you want me to change it like you mentioned, Though both are the same thing!